content/en/docs/tasks/traffic-management/egress/egress-kubernetes-services/index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Kubernetes Services for Egress Traffic
+description: Shows how to configure Istio  Kubernetes External Services.
+keywords: [traffic-management,egress]
+weight: 60
+---
+
+Kubernetes [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
+services and Kubernetes services with
+[Endpoints](https://kubernetes.io/docs/concepts/services-networking/service/#services-without-selectors)
+let you create a local DNS _alias_ to an external service.
+This DNS alias has the same form as the DNS entries for local services, namely
+`<service name>.<namespace name>.svc.cluster.local`. DNS aliases provide _location transparency_ for your workloads:
+the workloads can call local and external services in the same way. If at some point in time you decide to deploy the
+external service inside your cluster, you can just update its Kubernetes service to reference the local version. The workloads will continue to operate without any change.
+
+This task shows that these Kubernetes mechanisms for accessing external services continue to work with Istio.
+The only configuration step you must perform is to use a TLS mode other than Istio's
+[mutual TLS](/docs/concepts/security/#mutual-tls-authentication). The external services are not part of an Istio service
+mesh so they cannot perform the mutual TLS of Istio. You must set the TLS mode according to the TLS requirements of the
+external service and according to the way your workload accesses the external service. If your workload issues plain
+HTTP requests and the external service requires TLS, you may want to perform TLS origination by Istio. If your workload
+already uses TLS, the traffic is already encrypted and you can just disable Istio's mutual TLS.
```

Comment:
should we make a note here that in this case, what you 'd lose?  i assume telemetry data and possibly other l7 controls?

Reply:
@linsun Right, for TLS traffic the users would lose L7 telemetry and controls, but this topic (TLS traffic inside a service mesh) is a general Istio one, not related to the current task.